src/routes/Person.jsx

```diff
@@ -0,0 +1,118 @@
+import { DateTime } from "luxon";
+import Spinner from "../components/Spinner";
+import Swiper from "../components/Swiper";
+import PageLayout from "../components/layouts/PageLayout";
+import usePerson from "../hooks/usePerson";
+import usePersonCredits from "../hooks/usePersonCredits";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+
+const renderMovie = (cast) => {
+  const releseDate = DateTime.fromISO(cast.release_date).toFormat(
+    "dd LLL yyyy"
+  );
+  return (
+    <>
+      <div>
+        <Link to={`/movie/${cast.id}`}>
+          <img
+            src={`https://image.tmdb.org/t/p/w500/${cast.poster_path}`}
+            title={cast.title}
+            className="imageSlide rounded-xl max-h-56 min-w-[150px]"
+          />
+        </Link>
+
+        <div className="titleSlide text-base font-bold pt-6 px-2">
+          <Link to={`/movie/${cast.id}`} title={cast.title}>
+            {cast.title}
+          </Link>
+        </div>
+        <div className="subtitleSlide text-base px-2">{releseDate}</div>
+      </div>
+    </>
+  );
+};
+
+export default function Person() {
+  const [fullText, setFullText] = useState(false);
+  const person = usePerson();
+  const personCredits = usePersonCredits();
+
+  const briefFullText = (fullText) => {
+    return fullText ? "h-auto" : "h-72 overflow-hidden";
+  };
+
+  if (!person || !personCredits) return <Spinner />;
```

Comment:
Add indent after condition rendering.
It's an important point for components and better write this code with up and down indents.